### ISSUE TITLE
Split an item by different proportions

### ISSUE THREAD

**Issue by alexgleason** [cowbell/splittypie#8]
Hi,

First off, thanks for creating this! We were looking for a free-software app to track expenses in our housing cooperative. SplittyPie will work great for us.

Most of our expenses are split evenly, so this works for the majority of things. But rent is divided based on the size of our rooms, so we need more control over the proportions. Letting us type in dollar amounts or percentages next to each person would be awesome.

Technically we can work around this by separating each room into its own item, but that seems wrong. We are splitting the cost of rent, not individually renting rooms. To be clear, one person pays the property owner a single check each month, so the proper way to represent this is that rent was "split" with different proportions.

Thanks!

**Comment by tsubik** [comment]
I agree, that would be a fine enhancement. However, recently I literally have no time for any side projects. There is already a backlog of features waiting for implementation :smiley:
